x/nft/testutil/app_config.go

```diff
@@ -1,20 +1,108 @@
 package testutil
 
 import (
-	_ "embed"
-
 	"cosmossdk.io/core/appconfig"
-	_ "github.com/cosmos/cosmos-sdk/x/auth"
-	_ "github.com/cosmos/cosmos-sdk/x/auth/tx/module"
-	_ "github.com/cosmos/cosmos-sdk/x/bank"
-	_ "github.com/cosmos/cosmos-sdk/x/genutil"
-	_ "github.com/cosmos/cosmos-sdk/x/mint"
-	_ "github.com/cosmos/cosmos-sdk/x/nft/module"
-	_ "github.com/cosmos/cosmos-sdk/x/params"
-	_ "github.com/cosmos/cosmos-sdk/x/staking"
-)
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
once cli tests are migrated to mocks, we can remove these right?.

Reply:
It's not really clear to me how to migrate the cli tests to mocks, can you share some pointers? To me it seems that it will be a big-ish rewrite, will merge this and come to it when I have more ideas on how to make it happen

Reply:
So we have to re-write CLI integration tests such that we can finally remove the dependency on in-process Tendermint. We can achieve this one of two ways:

1. Mock Tendermint, i.e. create a super dumb and simple ABCI wrapper.
2. Mock the commands themselves, via outputs/inputs.

I'd rather go with option (1) as that'll be much less of a massive re-write. In fact, CLI tests wont have to change at all!

We spoke about this in Lisbon. When Marko is referring to mocks, he's referring to mocking Tendermint.